pset-iterators.js

```diff
@@ -66,6 +72,14 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const presidentsNamedJames = presidents.filter((president) => {
+	return president.president.includes('James')
+}).map((president) => {
+    return president.president
+})
+
+console.log(`Presidents with the first name of James are: ${presidentsNamedJames}.`)
+
```

Comment:
@cswebdesign Good Job here using the filter method in combination with the includes method to find the presidents whose names include James. One suggestion I see why you used the map to return only the president's name but in this problem we wanted you to return each president object. All you needed to do was use the filter method and that would've been enough.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L62)